dev-docs/analytics/automatad.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: analytics
+title: Automatad
+description: Automatad Prebid Analytics Adapter
+modulecode: automatadAnalytics
+gdpr_supported: false
```

Comment:
please update to `tcfeu_supported` -- this field has been changed. Thanks!

Reply:
This has been updated.